l10n_cn_fapiao/fapiao.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields
+
+
+class fapiao(models.Model):
+    _name = "fapiao"
+    _inherit = 'mail.thread'
+    _order = "fapiao_date desc"
+
+    fapiao_type = fields.Selection(
+        [('customer', 'Customer'), ('supplier', 'Supplier'),
+         ('customer_credit_note', 'Customer Credit note')],
+        'Fapiao Type', required=True, default='customer')
+    tax_type = fields.Selection(
+        [('13%', '13%'), ('17%', '17%'),
```

Comment:
@franksongfeng can you chec with Connie whether it should be better to have comodel?